chips/cc26x2/src/chip.rs

```diff
@@ -7,28 +7,21 @@ use rtc;
 use uart;
 
 pub struct Cc26X2 {
-    mpu: cortexm4::mpu::MPU,
```

Comment:
Seeing this, I think I probably should have included the `KernelUserlandBoundary` type here, and added a function to get it, rather than adding `cortexm4::syscall::SysCall::new()` to main.rs. I don't think it makes sense to ask board developers to choose the correct version when it is tied to the chip.

Thoughts on undoing this deletion and then calling `chip.mpu()` in the main.rs file?

Reply:
We decoupled the mpu from the chip structs because of an obstacle we encountered with Rust lifetimes: since `kernel_loop` already takes a mutable reference to the chip, Rust wouldn't allow us to pass `chip.mpu()` into `load_processes`.

Reply:
What if we make Chip not mutable? See https://github.com/tock/tock/compare/new-mpu-bradjc?expand=1